source/common/stream_info/stream_info_impl.h

```diff
@@ -437,26 +437,22 @@ struct StreamInfoImpl : public StreamInfo {
   SystemTime start_time_;
   MonotonicTime start_time_monotonic_;
   absl::optional<MonotonicTime> final_time_;
-
   absl::optional<Http::Protocol> protocol_;
-
-private:
-  absl::optional<uint32_t> response_code_;
-
-public:
-  absl::optional<std::string> response_code_details_;
-  absl::optional<std::string> connection_termination_details_;
   absl::InlinedVector<ResponseFlag, 4> response_flags_{};
   bool health_check_request_{};
   Router::RouteConstSharedPtr route_;
   envoy::config::core::v3::Metadata metadata_{};
   FilterStateSharedPtr filter_state_;
+
+private:
```

Comment:
Can we keep the order of the fields the same? This is a fairly common data structure and the layout would change if we-order - possibly for worse.

Reply:
The tests have a size check which passed: https://github.com/envoyproxy/envoy/blob/c6b8553313b818e981cad06d0d2a9d190df92c29/test/common/stream_info/stream_info_impl_test.cc#L44-L48, so I assumed the ordering was fine, but given they check a bunch of different sizes, I reverted back to the same ordering as before.  PTAL.